namespace cpp cc.service.lsp
namespace java cc.service.lsp

service LspService
{
  string getLspResponse(1: string request)
}